### ISSUE TITLE
Copular clause WH subject vs. predicate

### ISSUE THREAD

**Issue by nschneid** [UniversalDependencies/UD_English-EWT#514]
For WH-questions like "What's his name?" or "When was this?", a lot of the trees treat the WH-word as the predicate, as if there's question inversion. 

Is this correct? I thought our policy was to treat these as having a WH-subject and no inversion. (Due to the nature of the copula it's possible to paraphrase with either order.)

- 74 [EWT hits](https://universal.grew.fr/?custom=65ebab05e79e4)
- 37 [GUM hits](https://universal.grew.fr/?custom=65ebab2074aea)

**Comment by amir-zeldes** [comment]
My strategy for these, which is perhaps latently permeated by some early courses in Generative Grammar, is to transform the question into its answer and assign roles based on the mapped phrases in the answer. Taking the first example in GUM:

  * What do you think was the main reason for these experiences?

I construct the response:

  * The main reason for these experiences was X.

So I take it that "what" is a predicate, and "reason" is a subject. Naturally, it's also possible to paraphrase a response such as "X was the main reason", but that's not the first thing my brain did, so I assume that "reason" is nsubj. I've never done an IAA experiment specifically targeting this, but I imagine there are lots of symmetrical copula situations in languages with free word order where people do some kind of mental gymnastics like this to resolve ambiguous cases.

**Comment by nschneid** [comment]
It seems to me that "What"-questions often collapse the information structural distinction that determines subject versus predicate in declaratives where both are NPs:

- My biggest concern is the warranty. / The warranty is my biggest concern.
  * What is your biggest concern?
  * #What is the warranty?

With AdjP and PP predicates, on the other hand, it can be clearer whether the equivalent of a declarative subject vs. predicate is being questioned:
- My sweater is blue. ("Blue is my sweater." would be a highly noncanonical word order, even to the point of being archaic.)
  * What/nsubj is blue/root?
  * [What color/root] is your sweater/nsubj?
- My sweater is on the table. ("On the table is my sweater." is possible but not the canonical word order.)
   * What/nsubj is on the table/root?
   * Where/root is your sweater/nsubj? (at least this is the EWT policy; GUM has the noun as root in copular "where" sentences, with no subject!)

For the first case (two NPs), I'm skeptical that there's enough context to reliably guess which is the more likely information structure in context. I would vote for simply saying that "What" here is a subject and the word order resembles a declarative copular sentence.

**Comment by amir-zeldes** [comment]
> GUM has the noun as root in copular "where" sentences, with no subject!

Can you give an example? I can't see them in the query you linked above (could be an error)

> I would vote for simply saying that "What" here is a subject and the word order resembles a declarative copular sentence.

That doesn't sound right to me, I think if I had to pick just one structure, I would pick the opposite. The normal function of a copula "what" question is to request information about something, not to predicate things about an unknown "what". In plural cases, the agreement pattern is also determined by the lexical NP, e.g. "what are your reasons?", suggesting that "what" is probably not the subject. The word order is the expected one in English despite "what" not being the subject, due to the typical Germanic WH-fronting.

**Comment by amir-zeldes** [comment]
Another piece of evidence - person agreement:

  * what am I?
  *  * what is I?

**Comment by nschneid** [comment]
Good point about person agreement, though if we take the construction to be ambiguous, that is only useful for the small number of sentences that are copular questions about a personal pronoun.

I turned, as usual, to CGEL, which on p. 917 offers tests to diagnose the function of the WH-word in certain kinds of copular sentences: e.g. in "Who is editor of the magazine?", "who" has to be the subject. But others are ambiguous:

<img width="544" alt="image" src="https://github.com/UniversalDependencies/UD_English-EWT/assets/985263/96595bd9-48bd-4ccf-aae1-ed2638622de2">

I think "specifying" *be* refers to sentences like the ones we are talking about, where the declarative version of the copular clause identifies the value or identity of something, in contrast with "ascriptive" *be* where the copula introduces a property or role. So I guess CGEL is throwing up its hands on distinguishing the specifying ones based on formal criteria.

Would you agree that these WH-adverb ones should not be `advmod`? https://universal.grew.fr/?custom=65f4cd5154c7f

**Comment by amir-zeldes** [comment]
Yes, those are clearly errors, will fix.
